www/src/pages/blog/demo-day-2021-09.astro

```diff
@@ -0,0 +1,52 @@
+---
+import { Markdown } from 'astro/components';
+import BaseHead from '../../components/BaseHead.astro';
+import BlogHeader from '../../components/BlogHeader.astro';
+import BlogPost from '../../components/BlogPost.astro';
+
+let title = 'Astro Demo Day September Edition';
+let description = 'Astro September Demo Day was today and we had 4 amazing talks, including one with big announcements on the future direction of Astro.';
+let publishDate = 'September 20, 2021';
+let permalink = 'https://astro.build/blog/demo-day-2021-09';
+let lang = 'en';
+---
+
+<html lang={ lang ?? 'en' }>
+  <head>
+    <BaseHead title={title} description={description} canonicalURL={permalink} image="/assets/blog/demo-day-2021-09/social.png" />
+    <link rel="stylesheet" href={Astro.resolve('../../scss/blog.css')} />
+    <style>
+      .chat-messages {
+
+      }
+    </style>
+  </head>
+
+  <body>
+    <BlogHeader />
+    <BlogPost title={title} author="matthew" heroImage="/assets/blog/demo-day-2021-09/hero.png" publishDate={publishDate}>
+      <Markdown>
+        **Astro Demo Day** was today! We had **4** amazing talks, including one big announcement on the future direction of Astro. If you missed it that's really unfortunate, but good news! You can see the full event [on YouTube](https://www.youtube.com/watch?v=-ExcBJrXOd8) or watch the talks in any order by visiting the links below. So you'll never miss another demo day please [subscribe to our channel](https://www.youtube.com/channel/UCeFjmvZEK-MoefuYXptnX6A), or better yet [join our Discord](https://astro.build/chat) and get involved. Today's demos were:
```

Comment:
"If you missed it that's really unfortunate, but good news!" is hard to read. Maybe just: "If you missed the livestream, fear not!"

Reply:
"So you'll never miss another demo day please subscribe" is unnecessarily passive. 

Maybe: "Subscribe to the Astro YouTube channel to make sure you don't miss our next demo day."